docs/04-TechnicalInfrastructure.md

```diff
@@ -1 +1,323 @@
+    the directory `data/trashbin`.
+    
+```
+## Content of the /data/project/data/trashbin directory
+.
+|__ Tarload-XX1
+    |__ file.mnc
+    |__ file.mnc
+|__ Tarload-XX2
+    |__file.mnc
+```
+     
+- The MINC images that can be viewed via BrainBrowser in the imaging browser 
+    module are located under the `data/assembly` directory and organized by 
+    `CandID/Visit_label`. Whithin each of these visit labels, data are first 
+    organized by imaging type (`mri` or `pet` for example) and then by output 
+    type (such as `native` or `processed`). For example, a native T1W image for 
+    subject 123456's V1 visit will be located in 
+    `data/assembly/123456/V1/mri/native/Project_123456_V1_T1W_001.mnc`.
+    
+```
+## Content of the /data/project/data/assembly directory
+.
+|__ candid
+    |__ visit
+        |__ mri
+        |   |__ native
+        |   |   |__ Project_candid_visit_modality_number.mnc
+        |   |__ processed
+        |       |__ pipeline_name
+        |           |__ nativeFileName_output_number.mnc
+        |__ pet
+            |__ native
+                | Project_candid_visit_modality_number.mnc
+```
+
+- The screenshots displayed in the imaging browser module for each modality is 
+    stored within the `data/pic` folder and organized per candidates. 
+    
+```
+## Content of the /data/project/data/pic directory
+.
+|__ candid
+    |__ Project_candid_visit_modality_number_fileid_check.jpg
+    |__ Project_candid_visit_modality_number_fileid_check.jpg
+```
+    
+- Additionally, jiv images are produced by the imaging pipeline and are 
+    organized per candidates in the `data/jiv` folder.
+    
+```
+## Content of the /data/project/data/jiv directory
+.
+|__ candid
+    |__ Project_candid_visit_modality_number_fileid.header
+    |__ Project_candid_visit_modality_number_fileid.raw_byte.gz
+```
+    
+- Finally, processed incoming data or DTIPrep pipeline outputs are stored within 
+    the `data/pipelines` directory and organized per pipeline versions, 
+    candidates and visit labels. In addition, protocol files for automatic 
+    pipelines are saved in the `data/protocols` directory.
+    
+```
+## Content of the /data/project/data/pipelines directory
+.
+|__ DTIPrep
+    |__ DTIPrep_version
+        |__ candid
+            |__ visit
+                |__ mri
+                    |__ processed
+                        |__ DTIPrep_XML_protocol_name
+                            |__ file.mnc
+                            |__ file.nrrd
+
+## Content of the /data/project/data/protocols directory
+.
+|__ protocols
+    |__ DTIPrep
+        |__ Project_DTIPrep_XML_protocol.xml
+```
+
+## 4.2 Database infrastructure
+
+The database infrastructure is divided in six main components based on the 
+  workflow happening from native images insertion to quality control and 
+  ultimately insertion of processed datasets.  
+
+![overall_DB_structure](images/overall_DB_structure.png)
```

Comment:
diagrams are really good! there is just one small mistake for the type of CandID (`int(10)`), which appears to have taken on the datatype of CenterID (`tinyint(2)`)
i'm not sure how you made these diagrams, so if it is too much trouble then i would not worry about it